mappings/net/minecraft/client/render/animation/Animatior.mapping

```diff
@@ -0,0 +1,28 @@
+CLASS net/minecraft/unmapped/C_qrwoclyv net/minecraft/client/render/animation/Animatior
+	METHOD m_dcgpwvbl scale (DDD)Lnet/minecraft/unmapped/C_zviwfwgf;
+		ARG 0 x
+		ARG 2 y
+		ARG 4 z
+	METHOD m_fyhlfhpd getAnimationTimestamp (Lnet/minecraft/unmapped/C_mnzbondw;J)F
+		ARG 0 animation
+		ARG 1 millis
```

Comment:
Is this milliseconds?
If so, it seems like an odd choice, I'd expect either the full word or `ms`.

Reply:
ok

Reply:
clarified to `elapsedMillis`